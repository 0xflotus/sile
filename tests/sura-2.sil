\begin[papersize=a5,direction=RTL]{document}
\nofolios
\font[family=Amiri Quran,size=30pt]
\script[src=packages/linespacing]
\set[parameter=linespacing.method,value=fixed]
\set[parameter=linespacing.fixed.baselinedistance,value=200pt]
 الۤمۤ ۝١ 
 ذَٰلِكَ ٱلۡكِتَٰبُ لَا رَیۡبَۛ فِیهِۛ هُدࣰى لِّلۡمُتَّقِینَ ۝٢
 ٱلَّذِینَ یُؤۡمِنُونَ بِٱلۡغَیۡبِ وَیُقِیمُونَ ٱلصَّلَوٰةَ وَمِمَّا رَزَقۡنَٰهُمۡ یُنفِقُونَ ۝٣
 وَٱلَّذِینَ یُؤۡمِنُونَ بِمَاۤ أُنزِلَ إِلَیۡكَ وَمَاۤ أُنزِلَ مِن قَبۡلِكَ وَبِٱلۡءَاخِرَةِ هُمۡ یُوقِنُونَ ۝٤
 \end{document}