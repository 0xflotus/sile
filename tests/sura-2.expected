<tests/sura-2.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	350.3676
My 	75.4230
Set font 	Amiri Quran;30;400;;normal;;RTL
T	120 1347 120 1171 42	(الۤمۤ)
Mx 	341.5785
T	3	( )
Set font 	Amiri Quran;30;400;;normal;;LTR
T	1316	(١)
Mx 	303.1996
T	113	(۝)
Mx 	241.0567
Set font 	Amiri Quran;30;400;;normal;;RTL
T	76 1231 78 1228 1310 76 51	(ذَٰلِكَ)
Mx 	151.1404
T	77 478 1311 76 497 78 529 117 1348 531 101	(ٱلۡكِتَٰبُ)
Mx 	119.5981
T	700 76 698	(لَا)
Mx 	60.7951
T	111 76 879 117 875 76 52	(رَیۡبَۛ)
Mx 	20.9764
T	111 78 523 501 78 508	(فِیهِۛ)
Mx 	347.9115
My 	275.4230
T	71 140 503 77 525	(هُدࣰى)
Mx 	276.5342
T	76 927 926 78 511 377 79 497 77 827 117 1238 78 79 1228	(لِّلۡمُتَّقِینَ)
Mx 	267.7451
T	3	( )
Set font 	Amiri Quran;30;400;;normal;;LTR
T	1317	(٢)
Mx 	229.3662
T	113	(۝)
Mx 	172.0074
Set font 	Amiri Quran;30;400;;normal;;RTL
T	76 752 751 78 1121 377 79 1117 101	(ٱلَّذِینَ)
Mx 	98.6818
T	76 68 561 77 496 78 534 117 560 77 490	(یُؤۡمِنُونَ)
Mx 	20.9764
T	78 478 117 501 76 473 117 531 468 78 1061	(بِٱلۡغَیۡبِ)
Mx 	317.0033
My 	475.4230
T	76 68 561 77 535 501 78 511 77 490 76 70	(وَیُقِیمُونَ)
Mx 	247.5990
T	76 44 100 583 76 581 377 79 547 531 101	(ٱلصَّلَوٰةَ)
Mx 	207.4330
T	465 377 79 1141 78 1140 76 70	(وَمِمَّا)
Mx 	123.4535
T	117 533 77 526 1311 76 496 117 507 76 53 76 52	(رَزَقۡنَٰهُمۡ)
Mx 	51.6469
T	76 68 561 77 511 78 512 714 77 875	(یُنفِقُونَ)
Mx 	42.8578
T	3	( )
Set font 	Amiri Quran;30;400;;normal;;LTR
T	1318	(٣)
Mx 	4.4789
T	113	(۝)
[1] New page
Mx 	339.8549
My 	65.6525
Set font 	Amiri Quran;30;400;;normal;;RTL
T	76 752 751 78 1121 377 79 1117 101 76 70	(وَٱلَّذِینَ)
Mx 	266.7622
T	76 68 561 77 496 78 534 117 560 77 490	(یُؤۡمِنُونَ)
Mx 	228.1958
T	120 1363 76 802 78 794	(بِمَاۤ)
Mx 	176.2701
T	76 66 78 579 573 379 38	(أُنزِلَ)
Mx 	122.0007
T	76 527 117 501 76 531 381 40	(إِلَیۡكَ)
Mx 	72.9021
T	120 1363 76 534 76 70	(وَمَاۤ)
Mx 	20.9764
T	76 66 78 579 573 379 38	(أُنزِلَ)
Mx 	368.4193
My 	265.6525
T	536 78 534	(مِن)
Mx 	311.2754
T	76 1236 78 1233 117 494 76 507	(قَبۡلِكَ)
Mx 	212.5446
T	78 44 76 933 78 929 700 76 1313 117 698 468 78 1061 76 70	(وَبِٱلۡءَاخِرَةِ)
Mx 	172.7444
T	117 533 77 525	(هُمۡ)
Mx 	102.3582
T	76 68 561 77 496 78 507 561 77 490	(یُوقِنُونَ)
Mx 	93.5692
T	3	( )
Set font 	Amiri Quran;30;400;;normal;;LTR
T	1319	(٤)
Mx 	55.1903
T	113	(۝)
[2] End page
Finish

